spec/services/atmosphere/optimizer_spec.rb

```diff
@@ -13,7 +13,7 @@
   let!(:fund) { create(:fund) }
   let!(:openstack) { create(:openstack_with_flavors, funds: [fund]) }
   let!(:tmpl_of_shareable_at) { create(:virtual_machine_template, appliance_type: shareable_appl_type, \
-    tenant: openstack) }
+    tenants: [openstack]) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 16, 27 should be on its own line.